conf/taiga/docker-settings.py

```diff
@@ -107,3 +107,40 @@ def load_file(path):
 if os.getenv("SAML_AUTH_ENABLE").lower() == "true":
   INSTALLED_APPS += ["taiga_contrib_saml_auth"]
   SAML_AUTH = json.loads(os.getenv("SAML_AUTH_JSON_CONFIG") or 'null')
+
+#########################################
+## LDAP AUTH
+#########################################
+
+
+if os.getenv("LDAP_AUTH_ENABLE").lower() == "true":
+  INSTALLED_APPS += ["taiga_contrib_ldap_auth_ext"]
+  # TODO https://github.com/Monogramm/taiga-contrib-ldap-auth-ext/issues/16
+  LDAP_SERVER = os.getenv("LDAP_SERVER")
+  LDAP_PORT = int(os.getenv("LDAP_PORT"))
+
+  # Flag to enable LDAP with STARTTLS before bind
+  LDAP_START_TLS = os.getenv("LDAP_START_TLS")
```

Comment:
Underlying module expects a true boolean for this, so this conversion is needed
```suggestion
  LDAP_START_TLS = True if os.getenv("LDAP_START_TLS").lower() == "true" else False
```

Reply:
nice detail investigation